06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md

```diff
@@ -0,0 +1,10 @@
+# Setting up repository
+
+
+## Output
+- **Used**: GitHub
+- **Deliverable**: A repository for proposed project and at least one Pull Request
+
+
+## Link to my GitHub Repo
+https://github.com/andela-mugorji/Inverted-Index
```

Comment:
Hi
Thank for the feedback.
I have added content to my ReadMe and also used the Markdown equivalent to URLs, about naming my branch 'servant', that was just my imagination running wild(because the main branch is called 'master'). I have readjusted to branch naming convention.

I am reopening this pull request.